portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,86 @@
+:root {
+    --navy: #1b2d45;
+}
+
+html,
+body {
+    background-color: #f7f7f0;
+}
+
+a {
+    color: #007bff;
+    text-decoration: none;
+}
+
+a:hover {
+    color: #0056b3;
+}
+
+.row {
+    align-items: center;
+    /* border: 2px solid red; */
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    margin: 30px auto;
+    max-width: 1200px;
+    padding: 30px 5px;
+    width: 80%;
+}
+
+/* header styles */
+
+#header {
+    max-width: 800px;
+}
+
+#profile-img {
+    border-radius: 50%;
+    height: 200px;
+    object-fit: cover;
```

Comment:
nit: `object-fit` [isn't supported in IE](https://caniuse.com/#feat=object-fit). A typical workaround for this is to add a wrapper div for the image, and apply the `border-radius: 50%;` on that along with an `overflow: hidden;` attribute.